src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/SupplementaryEvidenceMapper.java

```diff
@@ -76,6 +77,21 @@ public SupplementaryEvidence map(
                 envelopeReferenceHelper.parseEnvelopeReferences(existingEnvelopeReferences)
             );
 
+            String existingReferences = updatedEnvelopeReferences
```

Comment:
`existingReferences`? Should not be `updatedReferences`?

Reply:
these are references already on the case

Reply:
in line 95 it adds new reference